index.html

```diff
@@ -0,0 +1,92 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Survey Form</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <!-- Page Title -->
+    <h1 id="title">freeCodeCamp Survey Form</h1>
+    <!-- Description -->
+    <p id="description">Thank you for taking the time to help us improve the platform</p>
+
```

Comment:
- [ ] Good job @oovillagran 😃, but I suggest you add `emoji's` to your comments so that the student understands the tone you are trying to communicate 😎. 
- [ ]  I also strongly suggest that in order to sound polite you need to be using words like `please`, `kindly`, I suggest, and `I highly recommend ...etc, and many other words you find properly. 👍